src/exercises/Congratulations.js

```diff
@@ -42,7 +43,7 @@ export default function Congratulations({
   if (username === undefined) {
     return <LoadingAnimation />;
   }
-
+  const small = "large";
```

Comment:
this looks like obfuscating the code?

Reply:
I agree, removed it now.

Reply:
Was leftover from testing speakbutton. It has been refactored now.